### PR TITLE
Add watch to webpack

### PR DESCRIPTION
**Pull request by AbrahamAlcaina**
I hope, it help in the issue 23

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: app/templates/gulpfile.js [hunk 1]
```diff
@@ -24,3 +24,8 @@ gulp.task('webpack:dev', ['clean'], function() {
     gulp.src(path.resolve(webpackConfigPath))
         .pipe(webpack.run());
 });
+
+
+gulp.task('build-cli-dev', ['webpack:dev'], function() {
+    gulp.watch(['stores/**/*.js'], ['webpack:dev']);
```
Comment (redonkulus) [inline app/templates/gulpfile.js]: can you add components folder too?

**Comment by redonkulus** [comment]
@AbrahamAlcaina this looks good, just need to add components to the watch list.

**Comment by AbrahamAlcaina** [comment]
Added

**Comment by redonkulus** [comment]
Thanks!

**Pull request merged**